### DISCUSSION TITLE
Disagreement between image updates and chart version

### DISCUSSION THREAD

**Discussion by ghost** [fluxcd/flux2#1745] [Q&A]
I'm looking at the image update to Git automation because I want to just be able to push tags, have a GitHub Action build and push the new image to the registry to trigger redeployment. (Is this not really GitOps, though, if tags can be deleted?)

I am [here](https://fluxcd.io/docs/guides/image-update/#configure-image-update-for-custom-resources), looking at the HelmRelease example manifest.

I understand that only the HelmRelease manifest will be affected by the image update?

The chart version will be unaffected by the image update, then?

Is it more appropriate to not use a Helm Chart then if I want to use this component, or should the image update not have anything to do with the chart version?

For example, if I am using a GitRepository, should I still need to bump the chart version for the changes to take effect?

**Comment by sch1ldkr0ete** [comment]
As far as I know when using a Helm chart stored in a Git repository you have to bump the chart version for the changes to take effect. If your chart is in a Helm repository then that is not the case and the changes take effect with no need for changing the chart version.

The chart version is unaffected by the image update. There is a proposal to automatically update Helm chart versions though: https://github.com/fluxcd/flux2/discussions/1665
